internal/testing/enginetest/enginetest.go

```diff
@@ -310,70 +311,70 @@ func RunTestModuleEngine_Call_Errors(t *testing.T, et EngineTester) {
 			input:  []uint64{0},
 			module: imported.Ctx,
 			fn:     imported.Exports[wasmFnName].Function,
-			expectedErr: `wasm runtime error: integer divide by zero
-wasm backtrace:
-	0: wasm_div_by`,
+			expectedErr: `wasm error: integer divide by zero
+wasm stack trace:
+	imported.wasm_div_by(i32) i32`,
 		},
 		{
 			name:   "host function that panics",
 			input:  []uint64{math.MaxUint32},
 			module: imported.Ctx,
 			fn:     imported.Exports[hostFnName].Function,
-			expectedErr: `wasm runtime error: host-function panic
-wasm backtrace:
-	0: host_div_by`,
+			expectedErr: `host-function panic (recovered by wazero)
+wasm stack trace:
+	imported.host_div_by(i32) i32`,
 		},
 		{
 			name:   "host function panics with runtime.Error",
 			input:  []uint64{0},
 			module: imported.Ctx,
-			fn:     imported.Exports[hostFnName].Function, // TODO: This should be a normal runtime error
-			expectedErr: `wasm runtime error: runtime error: integer divide by zero
-wasm backtrace:
-	0: host_div_by`,
+			fn:     imported.Exports[hostFnName].Function,
+			expectedErr: `runtime error: integer divide by zero (recovered by wazero)
+wasm stack trace:
+	imported.host_div_by(i32) i32`,
 		},
 		{
 			name:   "wasm calls host function that panics",
 			input:  []uint64{math.MaxUint32},
 			module: imported.Ctx,
 			fn:     imported.Exports[callHostFnName].Function,
-			expectedErr: `wasm runtime error: host-function panic
-wasm backtrace:
-	0: host_div_by
-	1: call->host_div_by`,
+			expectedErr: `host-function panic (recovered by wazero)
+wasm stack trace:
+	imported.host_div_by(i32) i32
+	imported.call->host_div_by(i32) i32`,
 		},
 		{
 			name:   "wasm calls imported wasm that calls host function panics with runtime.Error",
 			input:  []uint64{0},
 			module: importing.Ctx,
 			fn:     importing.Exports[callImportCallHostFnName].Function,
-			expectedErr: `wasm runtime error: runtime error: integer divide by zero
-wasm backtrace:
-	0: host_div_by
-	1: call->host_div_by
-	2: call_import->call->host_div_by`,
+			expectedErr: `runtime error: integer divide by zero (recovered by wazero)
+wasm stack trace:
+	imported.host_div_by(i32) i32
+	imported.call->host_div_by(i32) i32
+	importing.call_import->call->host_div_by(i32) i32`,
 		},
 		{
 			name:   "wasm calls imported wasm that calls host function that panics",
 			input:  []uint64{math.MaxUint32},
 			module: importing.Ctx,
 			fn:     importing.Exports[callImportCallHostFnName].Function,
-			expectedErr: `wasm runtime error: host-function panic
-wasm backtrace:
```

Comment:
I would prefer retaining the stack index (0, 1, 2) as afaik all most all programming languages produces with such indexes and I can assume we are familiar with it

Reply:
oh well I told a lie :D maybe I'm fine without it 😄

Reply:
I probably should have mentioned I'm aiming for even more go'like when we get line numbers etc from dwarf. Let's let this ride a bit? 

note Aiming to be similar but easy to understand that it is wasm :D being too close can be confusing, but we don't have the luxury of enough metadata to do that :D